src/ubuntu/17.10/Dockerfile

```diff
@@ -14,7 +14,9 @@ RUN apt-get update \
         cmake \
         liblldb-3.9-dev \
         lldb-3.9 \
+	python-lldb-3.9 \
```

Comment:
It looks like you are using tabs for indents instead of spaces.  Spaces should be used as they are consistent across editors (e.g. different tab settings)

Reply:
Done. I hate tabs.